packages/stacked_generator/pubspec.yaml

```diff
@@ -7,15 +7,18 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
+  flutter:
+    sdk: flutter
+    
   build: ^2.0.0
   source_gen: ^1.0.0
   analyzer: ^1.3.0
   path: ^1.6.4
   # logger: ^0.9.0
   recase: ^4.0.0-nullsafety.0
   stacked:
-    ^2.0.0
-    # path: ../stacked
+    # ^2.0.0
+    path: ../stacked
```

Comment:
You have to change back to non-path dependency for me to release

Reply:
Done. please check